rtengine/imageio.cc

```diff
@@ -83,6 +85,43 @@ Glib::ustring to_utf8 (const std::string& str)
     }
 }
 
+unsigned short addSat (unsigned short a, unsigned short b)
```

Comment:
Isn't there an GCC intrinsic for saturated addition? I think it was SSE or MMX. In any case, this probably wants to go into a separate header otherwise. Maybe `rt_math.h`?